### PR TITLE
Use new trsv substitution template

### PR DESCRIPTION
**Pull request by tfalders**
Updates rocSOLVER to reflect changes in https://github.com/ROCmSoftwarePlatform/rocBLAS-internal/pull/664.

### CODE REVIEW

File: library/src/include/rocblas.hpp [hunk 1]
```diff
@@ -1449,49 +1449,42 @@ rocblas_status rocblasCall_symm_hemm(rocblas_handle handle,
...
+                                rocblas_int offset_A,
+                                rocblas_int lda,
+                                rocblas_stride stride_A,
+                                U B,
+                                rocblas_int offset_B,
+                                rocblas_int ldb,
+                                rocblas_stride stride_B,
+                                rocblas_int batch_count,
+                                rocblas_int* g_unique_row,
+                                rocblas_int* g_completed_sec,
```
Comment (cgmb) [inline library/src/include/rocblas.hpp:1472]: That's an unfortunate name. `g_*` is a naming convention often used for global variables. Oh well. I suppose there's not much we can do given that's what it's called.

**Review by cgmb — APPROVED** [review]
Should wait for the tests, but LGTM. 👍

File: library/src/include/rocblas.hpp (hunk 1 above)
Comment (TorreZuk) [reply library/src/include/rocblas.hpp:1472]: This is your template you should be able to call your variables whatever you want.   Ask @daineAMD and he could rename the rocBLAS side too, surprised I didn't notice it there as not sure what the g_ stands for if not global.

Comment (daineAMD) [reply library/src/include/rocblas.hpp:1472]: global as opposed to shared memory, but I'll make a change as I understand the mistake.

**Review by jzuniga-amd — APPROVED** [review]
LGTM!

**Comment by cgmb** [comment]
I think we can merge once the tests finish. The name can be updated later if rocblas changes it to something else.

> A variable by any other name would be just as sweet, if it compiled.

~ William ShakeCPPeare

**Pull request merged**